app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,36 @@
+class ReservationsController < ApplicationController
+  def new
+    #@reservation = Reservation.new
+    #@cart.data.each do |item_id, quantity|
+      #item = Item.unscoped.find(item_id.to_i)
+      #item.add_quantity(quantity)
+      #@reservation.items << item
+    #end
+  end
+
+  def create
+    #if current_user
```

Comment:
Incorrect indentation detected (column 4 instead of 6).<br>Missing space after #.